source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,348 @@
+.. _realm-sync-migrate-modes:
+
+=========================
+Migrate Device Sync Modes
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 3
+   :class: singlecol
+
+Atlas App Services Apps can have one of two Device Sync modes: Partition-Based Sync
+and Flexible Sync. Partition-Based Sync is an older mode and you should consider
+migrating to Flexible Sync.
+
+Migrating an App that uses Partition-Based Sync to Flexible Sync is an automatic
+process. Other than upgrading the SDK version, migrating Sync modes doesn't require 
```

Comment:
Is upgrading the SDK required? Seems like only if it is old-ish. As I read the sentence, it implies one must upgrade. If not, perhaps: Other than possibly needing to upgrade the SDK version...

Reply:
It is required. There's some automatic logic added to the SDK to enable the PBS client to communicate with the Flexible Sync backend. SDK versions older than the (TBD) version that includes the logic simply won't be able to communicate with the backend.